### PR TITLE
Add a Rewrap control

### LINKED ISSUE #3: Rewrap molecules
Add a `Control` algorithm to rewrap molecules inside the unit cell, using the molecule center of mass.

### PR DESCRIPTION
**Pull request by Luthaf**
Fix #3 =)

### CODE REVIEW

File: src/core/src/sim/md/controls.rs [hunk 1]
```diff
@@ -181,6 +181,26 @@ impl Control for RemoveRotation {
     }
 }
 
+
+/******************************************************************************/
+/// Rewrap all molecules inside the unit cell
```
Comment (g-bauer) [inline src/core/src/sim/md/controls.rs]: Maybe we can add that not all atoms of a molecule will lie within the unit cell.
```rust
/// Rewrap all molecules' centers of mass to lie within the unit cell. Atoms of a molecule may lie outside of the cell.

**Comment by g-bauer** [comment]
This will control the COMs of molecules to lie within the simulation cell, right? Atoms of molecules can still be found outside the cell. Maybe we can make that a bit clearer in the docstring? See my suggestion.

**Comment by Luthaf** [comment]
Yep, I'll improve the wording. This also needs some docs.

File: doc/src/input/md.md [hunk 2]
```diff
@@ -184,3 +184,5 @@ controls = [
 
 - The `RemoveTranslation` control removes the global system rotation;
 - The `RemoveRotation` control removes the global system translation.
+- The `Rewrap` control rewrap all molecules' centers of mass to lie within the
```
Comment (g-bauer) [inline doc/src/input/md.md]: rewrap**s**

Comment (Luthaf) [reply doc/src/input/md.md]: -_- ... do you know a good grammar checker for english? I really need one installed.

Comment (g-bauer) [reply doc/src/input/md.md]: Mh no not really...although I get constantly spammed with ads from [grammarly](https://www.grammarly.com) but I have no idea how/if it works with github :)

**Comment by g-bauer** [comment]
Ehm...what went wrong? `An unknown error occurred` is not very helpful.

**Comment by Luthaf** [comment]
That happens sometimes, for no clear reason. I restarted the build!

**Pull request merged**